ttnn/cpp/ttnn/operations/moreh/moreh_getitem/device/moreh_getitem_device_operation.cpp

```diff
@@ -34,8 +34,8 @@ void MorehGetItemOperation::validate_inputs(
         auto index_layout = index_tensor.get_layout();
         if (index_layout == Layout::ROW_MAJOR) {
             TT_FATAL(index_shape.rank() == 1, "Index tensor must be 1D for ROW_MAJOR layout!");
-        } else if (index_layout == Layout::TILE) {
-            TT_FATAL(index_shape.rank() == 5, "Index tensor must be 5D for TILE layout!");
+        } else {
```

Comment:
Would it be better to remove the empty else clause?

Reply:
Someone might be confused about whether omitting the `else` block was intentional or just a mistake, so I left the empty block.